### ISSUE TITLE
Can I get training model inside of custom callback function ? 

### ISSUE THREAD

**Issue by tempdeltavalue** [UKPLab/sentence-transformers#2413]
Generally this issue copy-paste of that:
https://github.com/UKPLab/sentence-transformers/issues/458

So I want something like in tf.keras, to be able get model every epoch (for ex.) and running/logging my metrics.. is it possible ?
https://stackoverflow.com/questions/65302030/access-loss-and-model-in-a-custom-callback

Thank you.

**Comment by tempdeltavalue** [comment]
@tomaarsen 
Thank you !
